src/appMain/signal_handlers.cc

```diff
@@ -31,7 +31,7 @@
 */
 
 #include "signal_handlers.h"
-#include "life_cycle.h"
+#include "./life_cycle.h"
```

Comment:
@AByzhynar why are u using relative path?
It's make your code fragile.

Reply:
@EKuliiev According to our rules folder which contain current file should be specified. Therefore: 
to follow the rules I need to write `#include "./file.h" ` instead of `#include "file."`

Reply:
@AByzhynar appMain ... I do not like this name appMain.
As for me u can leave this part as it was before "file.h"
There're in appMain dir too few files to pollute include directory search.
This is my opinion.

Reply:
@EKuliiev I agree with you but we have to follow our rules.

Reply:
@EKuliiev fixed in 595cee4db06641f9fc015157dc8f879958c299d2